FE/components/molecules/Button/Button.tsx

```diff
@@ -1,10 +1,10 @@
-import { ReactElement, MouseEventHandler } from 'react';
+import { ReactElement, ButtonHTMLAttributes, MouseEventHandler } from 'react';
 import styled from 'styled-components';
 import { Text } from '@atoms';
 
-interface EachButtonProps {
+interface EachButtonProps extends ButtonHTMLAttributes<HTMLButtonElement> {
   title: string;
-  func: MouseEventHandler;
+  func?: MouseEventHandler;
   width?: string;
 }
```

Comment:
```suggestion
  func?: MouseEventHandler;
  width?: string;
  type?: string;
}
```
props가 추가되어서 필요할 것 같습니다

Reply:
아 이게 저도 처음에 props에 추가해서 하려했는데 `extends ButtonHTMLAttributes<HTMLButtonElement> ` 이 구문으로 자동으로 생기고 저희가 다시 props를 선언하게 되면 에러가 나더라고요!!

Reply:
와 저걸  extends하면 자동으로 타이핑이 되는 모양이네요 한번 찾아보겠습니다!